opm/simulators/wells/RateConverter.hpp

```diff
@@ -493,11 +493,13 @@ namespace Opm {
 
                     // sum p, rs, rv, and T.
                     double hydrocarbonPV = pv_cell*hydrocarbon;
-                    pv += hydrocarbonPV;
-                    p += fs.pressure(FluidSystem::oilPhaseIdx).value()*hydrocarbonPV;
-                    rs += fs.Rs().value()*hydrocarbonPV;
-                    rv += fs.Rv().value()*hydrocarbonPV;
-                    T += fs.temperature(FluidSystem::oilPhaseIdx).value()*hydrocarbonPV;
+		    if (hydrocarbonPV > 0) {
```

Comment:
I think it is necessary not to trigger assertion failure when running without hydrocarbon phases.  It is similar to https://github.com/OPM/opm-simulators/pull/1961 . It is okay to be this way in this PR.